lib/default-theme/styles/config.styl

```diff
@@ -3,6 +3,7 @@ $accentColor = #3eaf7c
 $textColor = #2c3e50
 $borderColor = #eaecef
 $codeBgColor = #282c34
+$codeLang = js ts html md vue css sass scss less stylus go java c sh yaml py
```

Comment:
May not put this after `// colors`

Reply:
oh I'm sorry, I didn't see this until now. I can put this after `// code`, if you don't mind the additional commit. @ulivz do you have any thoughts about this? Should I leave it as is?